### ISSUE TITLE
insufficient contrast in PyPI colours

### ISSUE THREAD

**Issue by brainwane** [pypa/warehouse#3334]
From #2417:

> I like the `pip install xx` in the header, but because of the background color it doesn't really stand out and is easy to miss. I would suggest a more contrasting color?

Marking `accessibility` in case we aren't hitting a11y standards for contrast.

**Comment by brainwane** [comment]
I ran [an automated check](http://wave.webaim.org/report#/https://pypi.org) on the accessibility of pypi.org and found [several instances of insufficient contrast](http://wave.webaim.org/report#/https://pypi.org/project/exec-helpers/) by WCAG standards.

The  `pip install` box, the "Latest version" box, the "Get Help"/"About PyPI"/"Contributing to PyPI"/"Using PyPI" links in the footer, and some fixed-width text in the README were all marked as having insufficient contrast [per this guideline](http://webaim.org/standards/wcag/checklist#sc1.4.3).

I also saw multiple complaints about low-contrast parts of the new design, in particular "low-contrast pale blue text and icons on blue background", from developers [on python-list](https://mail.python.org/pipermail/python-list/2018-March/732182.html).

This accessibility issue is present in enough of the site, and affects enough users, that I believe we should not shut down legacy PyPI until we address it in Warehouse.

**Comment by nlhkabu** [comment]
Thanks @brainwane- assigning myself - will look into asap
